apps/minifront/src/components/tx-details/index.tsx

```diff
@@ -16,7 +16,7 @@ export const TxDetailsErrorBoundary = () => {
 export const TxDetails = () => {
   const { hash } = useParams<{ hash: string }>();
 
-  const txInfo = useTransactionInfo(hash!);
+  const txInfo = useTransactionInfo(undefined, hash!);
```

Comment:
`use[Name]()` hooks take an optional `options` object as their first argument. We'll leave it undefined here since we only want to pass arguments (in this case, `hash!`) directly to the `fetch` function.

Unfortunately, I couldn't figure out how to get TypeScript to play nice with adding a `fetchArgs` property to the options object, which would have been a nicer/more consistent DX. So instead, the arguments to the `fetch` arguments are any arguments that come after the options object.